docs/querying/basics.md

```diff
@@ -281,33 +316,33 @@ PromQL supports line comments that start with `#`. Example:
 
 ### Staleness
 
-When queries are run, timestamps at which to sample data are selected
-independently of the actual present time series data. This is mainly to support
+The timestamps at which to sample data, during a query, are selected
+independently of the actual present time series data. This method principally supports
 cases like aggregation (`sum`, `avg`, and so on), where multiple aggregated
-time series do not exactly align in time. Because of their independence,
+time series do not precisely precisely in time. Because of their independence,
 Prometheus needs to assign a value at those timestamps for each relevant time
-series. It does so by simply taking the newest sample before this timestamp.
+series. It does so by taking the newest sample before this timestamp.
 
 If a target scrape or rule evaluation no longer returns a sample for a time
-series that was previously present, that time series will be marked as stale.
-If a target is removed, its previously returned time series will be marked as
-stale soon afterwards.
+series that was previously present, this time series will be marked as stale.
+If a target is removed, the previously retrieved time series will be marked as
+stale soon after removal.
 
 If a query is evaluated at a sampling timestamp after a time series is marked
-stale, then no value is returned for that time series. If new samples are
-subsequently ingested for that time series, they will be returned as normal.
+as stale, then no value is returned for that time series. If new samples are
+subsequently ingested for that time series, they will be returned as expected.
 
-If no sample is found (by default) 5 minutes before a sampling timestamp,
-no value is returned for that time series at this point in time. This
-effectively means that time series "disappear" from graphs at times where their
-latest collected sample is older than 5 minutes or after they are marked stale.
+A time series will go stale when they're no longer exposed / the target no
+longer exists. This effectively means that time series "disappears" from graphs 
+at the times of their latest collected sample, and they will not show up in queries
+over a time interval they are marked stale.
```

Comment:
Both "target no longer exists" and "5 minutes" behaviours are still in Prometheus, although the latter is seen mostly with targets that set explicit timestamps.
But we should document both, not erase one.

Reply:
i've removed the change

Reply:
updated with your suggestion please take a look